CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Addressed bug with iOS mobile push notifications always being set to critical by @imtoori and @joeyorlando ([#1646](https://github.com/grafana/oncall/pull/1646))
```

Comment:
Not sure if we should include this to changelog, user settings are not released yet so this is not user-facing change, right?

Reply:
Nvm, this does actually make sense even if user settings are not released yet in the mobile app (user would need to update the backend to get this working correctly)